Searching/Linear Search/LinearSearch.js

```diff
@@ -0,0 +1,31 @@
+function linear_search(arr, searchkey)
+{
+  for(let i=0;i<n;i++)
+  {
+    if(arr[i]===searchkey)
+    {
+      return(i);
+    }
+    else
+    {
+      return(null);
```

Comment:
Instead of returning null, we can return -1, to ensure consistent return type(integer) of this function.

Reply:
okay